schema/registers.json

```diff
@@ -84,8 +84,9 @@
             "type": "string"
         },
         "converter": {
-            "description": "Specifies a custom converter method used to parse or format the payload value in the high-level interface.",
-            "type": "string"
+            "description": "Specifies whether a custom converter will be used to parse or format the payload value.",
```

Comment:
What happens if the user only wants to define one of the methods? Will they need to manually delete the reference of the "partial" in the respective method?

Reply:
If they specify a converter, they always need to implement both methods (see above).